ci/stemcell/light/pipeline.yml

```diff
@@ -2,72 +2,142 @@
 groups:
   - name: light-gce-stemcells
     jobs:
-      - ubuntu-trusty-stemcell
+      - ubuntu-trusty-stemcell-latest
+      - ubuntu-trusty-stemcell-3312
+      - ubuntu-trusty-stemcell-3263
+
+shared:
```

Comment:
It's not a [Concourse](https://concourse.ci/pipelines.html#table-of-contents) key — the only top-level keys that Concourse has are `resources`, `resource_types`, `jobs`, and `groups`.

Rather, this is a generic YAML key that we use to place YAML anchors that we re-use throughout the pipeline. We could have just as easily called it `anchors:` or whatever.

If you prefer, we'd be happy to inline the anchors and remove the `shared` key.